twilio/help.md

```diff
@@ -1,6 +1,6 @@
 # Description
 
-[Twilio](https://www.twilio.com/) is a cloud communications platform for building SMS, Voice & Messaging applications
+[Twilio](https://www.twilio.com/) is a cloud communications platform for building SMS and Voice & Messaging applications
```

Comment:
Should this be `building SMS, Voice, and Messaging applications`?

Reply:
tbh idk, that sounds good to me though